processing/src/main/java/io/druid/query/groupby/RowBasedColumnSelectorFactory.java

```diff
@@ -410,19 +418,25 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
         @Override
         public double getDouble()
         {
-          return row.get().getMetric(columnName).doubleValue();
+          return DimensionHandlerUtils.nullToZeroDouble(row.get().getMetric(columnName)).doubleValue();
```

Comment:
Shouldn't this method throw exception, if the value is null, and SQL-compatible handling is on? (same below)